src/index.js

```diff
@@ -15,24 +23,44 @@ const todoItems = [{
   index: 1,
 }];
 
+// function dropzoneSet() {
+//   const ul = document.getElementById('js-todo-list');
+//   ul.addEventListener(dragenter, (e) => { ulDragEnter(e), false; });
+//   ul.addEventListener(dragleave, (e) => {ulDragLeave(e), false});
+//   ul.addEventListener(dragover, (e) => {ulDragOver(e), false});
+//   ul.addEventListener(drop, (e) => {itemDrop(e), this});
+// }
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. 

![image](https://user-images.githubusercontent.com/31897434/77977676-3d0dec80-72f8-11ea-8176-dc41cb8135b6.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643